pandas/core/generic.py

```diff
@@ -1062,23 +1062,51 @@ def _indexer(self):
             # add to our internal names set
             cls._internal_names_set.add(iname)
 
-    def get(self, key, default=None):
+    def get(self, key, default=None, axis=None):
         """
         Get item from object for given key (DataFrame column, Panel slice,
-        etc.). Returns default value if not found
+        etc.) along the given axis. Returns default value if not found
 
         Parameters
         ----------
         key : object
+        default : object, optional
+            Value to return if key is not present
+        axis : int or None
+            The axis to filter on. By default this is the info axis. The "info
+            axis" is the axis that is used when indexing with ``[]``. For
+            example, ``df = DataFrame({'a': [1, 2, 3, 4]]})
 
         Returns
         -------
-        value : type of items contained in object
+        value : type of items contained in object, or default
         """
+        # special case (GH 5652)
```

Comment:
not needed, this just generates a ``ValueError`` which by definition returns ``default``